File: Reto2/Reto2Angular/src/app/models/hospital.ts
export class Hospital {
  idHospital: number;
  nombre: string;
  antiguedad: number;
  area: number;
  idDistrito: number;
  idSede: number;
  idGerente: number;
  idCondicion: number;
  fexhaCreacion:Date;
  id: any;
  constructor(
  
  ) {
    this.idHospital = 0;
    this.nombre ="";
    this.antiguedad = 0;
    this.area = 0;
    this.idDistrito = 0;
    this.idSede = 0;
    this.idGerente = 0;
    this.idCondicion =0;
    this.fexhaCreacion=new Date;
  }
}

File: Reto2/Microservicios/Monolito/src/main/java/Reto/Monolito/entity/Hospital.java
package Reto.Monolito.entity;

import javax.persistence.*;

import lombok.*;

import java.util.Date;
@Data
@Entity
@Table(name="Hospital")
@Setter
@Getter
@AllArgsConstructor
@NoArgsConstructor
public class Hospital {
    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    @Column(name = "idHospital")
    private Long idHospital;


    @Column(name = "nombre" ,length = 255)
    private String nombre;

    @Column(name = "Antiguedad")
    private Integer antiguedad;

    @Column(name = "Area")
    private Float area;
    @OneToOne
    @JoinColumn(name = "idDistrito")
    private Distrito distrito;

    @OneToOne
    @JoinColumn(name = "idSede")
    private Sede sede;

    @OneToOne
    @JoinColumn(name = "idGerente")
    private Gerente gerente;

    @OneToOne
    @JoinColumn(name = "idCondicion")
    private Condicion condicion;
    @Column(name = "Fecha creacion")
    @Temporal(TemporalType.TIMESTAMP)
    private Date date;


}
